dex-it/src/test/scala/com/wavesplatform/it/sync/BouncingBalancesTestSuite.scala

```diff
@@ -52,152 +54,152 @@ class BouncingBalancesTestSuite extends WsSuiteBase {
-      wsc.clearMessages()
+        assertChanges(wsc)(
+          Map(Waves -> WsBalances(4949949.997, 0.003)), // Fee for order
+          Map(doggyCoin -> WsBalances(0, 1000000))
+        )(WsOrder.fromDomain(LimitOrder(bobOrder)))
+        wsc.clearMessages()
 
-      dex1.api.orderStatusByAssetPairAndId(bobOrder).status shouldBe Status.Accepted
+        dex1.api.orderStatusByAssetPairAndId(bobOrder).status shouldBe Status.Accepted
 
-      withClue("After rollback order should not be cancelled and balances should not be decreased\n") {
-        wavesNode1.asyncApi.rollback(heightInitial, returnTransactionsToUtx = true) // true as on Node
-        wavesMinerNode.api.rollback(heightInitial, returnTransactionsToUtx = true) // true as on Node
-        eventually {
-          wavesNode1.api.currentHeight shouldBe >=(heightInitial)
-        }
+        withClue("After rollback order should not be cancelled and balances should not be decreased\n") {
+          wavesNode1.asyncApi.rollback(heightInitial, returnTransactionsToUtx = true) // true as on Node
+          wavesMinerNode.api.rollback(heightInitial, returnTransactionsToUtx = true) // true as on Node
+          eventually {
+            wavesNode1.api.currentHeight shouldBe >=(heightInitial)
+          }
 
-        wavesNode1.api.waitForHeight(heightSecondTransfer)
-        wavesNode1.api.waitForHeightArise() // See WavesFork
-        dex1.api.waitForOrderStatus(bobOrder, Status.Accepted)
-
-        //        wsc.messages.filter {
-        //          case _: WsPingOrPong => false
-        //          case _ => true
-        //        } shouldBe empty
-
-        // Relates DEX-1099
-        dex1.api.getTradableBalanceByAssetPairAndAddress(bob, AssetPair(doggyCoin, Waves)) should matchTo(Map[Asset, Long](
-          Waves -> 494994999700000L
-        ))
-      }
+          wavesNode1.api.waitForHeight(heightSecondTransfer)
+          wavesNode1.api.waitForHeightArise() // See WavesFork
+
+          dex1.api.waitForOrderStatus(bobOrder, Status.Accepted)
 
-      wsc.close()
+          //        wsc.messages.filter {
+          //          case _: WsPingOrPong => false
+          //          case _ => true
+          //        } shouldBe empty
+
+          // Relates DEX-1099
+          dex1.api.getTradableBalanceByAssetPairAndAddress(bob, AssetPair(doggyCoin, Waves)) should matchTo(Map[Asset, Long](
+            Waves -> 494994999700000L
+          ))
+        }
+      }
     }
 
     "multiple orders test" in {
-      val aliceWsc = mkWsAddressConnection(alice, dex1)
-      val bobWsc = mkWsAddressConnection(bob, dex1)
-
-      val heightInitial = wavesNode1.api.waitForHeightArise()
-      val aliceBalance1 = dex1.api.getTradableBalanceByAssetPairAndAddress(alice, wavesUsdPair)
-      val bobBalance1 = dex1.api.getTradableBalanceByAssetPairAndAddress(bob, wavesUsdPair)
+      Using(mkWsAddressConnection(alice, dex1)) { aliceWsc =>
+        Using(mkWsAddressConnection(bob, dex1)) { bobWsc =>
```

Comment:
Let's use `Using.Manager` to have less indentation: https://stackoverflow.com/a/55440868/4050580